wolox-books/src/app/screens/login/login.component.html

```diff
@@ -1 +1,15 @@
-<p>login works!</p>
+<main class="main-container">
+    <form [formGroup]="form" class="form-container">
+      <app-form (formSubmitted)="submitForm()" [isValid]="form.valid" redirect="../sign-up" submitLabel="Login" redirectLabel="Sign Up">
```

Comment:
Shouldn't the `form` be inside the `app-form` component?

Reply:
The only task I do on `app-form` is emit the SubmitForm event, while on Login component, I submit using the form values.